presentation/backend/src/presentation-backend/NativePlatform.ts

```diff
@@ -181,21 +181,10 @@ export const createDefaultNativePlatform = (props: DefaultNativePlatformProps):
       return this.handleVoidResult(this._nativeAddon.clearRulesets());
     }
     public async handleRequest(db: any, options: string) {
-      const requestGuid = this.handleResult(this._nativeAddon.queueRequest(db, options)).result;
-      return new Promise((resolve: (result: NativePlatformResponse<any>) => void, reject) => {
-        const interval = setInterval(() => {
-          const pollResult = this._nativeAddon.pollResponse(requestGuid);
-          if (pollResult.error) {
-            if (pollResult.error.status !== IModelJsNative.ECPresentationStatus.Pending) {
-              reject(new PresentationError(this.getStatus(pollResult.error.status), pollResult.error.message));
-              clearInterval(interval);
-            }
-            return; // ignore 'pending' responses
-          }
-          resolve(this.createSuccessResponse(pollResult));
-          clearInterval(interval);
-        }, 20);
-      });
+      const result = await this._nativeAddon.handleRequest(db, options);
```

Comment:
It looks like editing this file requires you to run `rush change`. Please also make sure you have run lint, extract-api, cover, etc.